testing/testcafe/tests/dataGrid/validation.ts

```diff
@@ -0,0 +1,45 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector } from 'testcafe';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+import DataGrid from '../../model/dataGrid';
+import { getData } from './helpers/generateDataSourceData';
+
+fixture.disablePageReloads`Validation`
+  .page(url(__dirname, '../container.html'));
+
+test('Validation popup screenshot', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const dataGrid = new DataGrid('#container');
+
+  await t
+    .maximizeWindow()
+    .click(dataGrid.getDataCell(0, 0).element)
+    .pressKey('ctrl+a backspace enter')
+    // act
+    .expect(await takeScreenshot('validation-popup', dataGrid.element))
```

Comment:
It's better to add the extension to the screenshot name:

```suggestion
    .expect(await takeScreenshot('validation-popup.png', dataGrid.element))
```